### DISCUSSION TITLE
Planning feature

### DISCUSSION THREAD

**Discussion by Robson1352** [kimai/kimai#2377] [General]
Hi there,
I am writing to you as I was thinking to use KIMAI for time tracking and planning for my projects, however, I was wondering if there is possibility to add some extra feature to it, in terms of time planner. What I mean is, I would like to create timesheet for planning time for my project and later compare it to actual time spend on it, it would help me with time management for future timing of my project management. This would add another dimension for timing analysis of planned vs real time spent on projects. Not sure if that sort of stuff is possible to implement. 
I would be grateful for any opinion in this matter.
Thx.

**Comment by kevinpapst** [comment]
Customers,. Projects and Activities have both a monetary and time budget. You have a progressbar while tracking Tims on these projects. 

More planning features are available in the [Task Management Plugin ](https://www.kimai.org/store/task-management-bundle.html) which can be tested in the plugins demo: http://demo-plugins.kimai.org

And the reports are enhanced in the future, Kimai is not feature complete...

**Reply by Robson1352** [reply]
Thx
